tap/extensions/http/main.go

```diff
@@ -189,9 +180,9 @@ func (d dissecting) Analyze(item *api.OutputChannelItem, resolvedSource string,
 	reqDetails["queryString"] = mapSliceRebuildAsMap(reqDetails["_queryString"].([]interface{}))
 
 	if resolvedDestination != "" {
-		service = SetHostname(service, resolvedDestination)
+		service = resolvedDestination
```

Comment:
@mertyildiran 

Not related to your change explicitly but it feels weird that same field (service) got the value from "resolvedSource" or "resolvedDestination" - those should be opposite values (like if request came from A -> B, the source will be A and the destination will be B, in this simple example request the service should be probably "B" because the request issued to him)..
Maybe I miss something or is it really mistake?

Reply:
@gadotroee the logic is like that for ages. Actually saying a traffic log belongs to a service is in accurate. Because that's a binary event that involves two services. But the design was like that since the very beginning.

We try to show a name instead of an IP with that `service` field, if destination is resolved we show destination. If destination is not resolved, we fallback to source. That's why the logic is like that.

Reply:
I understand you didn't change the behavior in this commit so I'm approving it because this fix is good but I do think that original behavior (maybe a long time ago) was to show the "SERVICE" (which is the target part) if it is not resolved we should (IMO) show the ip because it is more important to be consistent on what we see in this place.